### ISSUE TITLE
Product Dashboard not displaying any data in Commit section

### ISSUE THREAD

**Issue by sandilyasd** [capitalone/Hygieia#2347]
Hello, 

The Subversion Commit Data which is getting displayed in the  Team Dashboard is not available in Product dashboard.

Could you please help me with this?

**Comment by sandilyasd** [comment]
Hello,

I am using the bamboo collector to get information to Build Widget.
Both repo widget and build widget are configured but i didn't see any data in commit however i am seeing the data in Build tab.

Could you please help me here?

**Comment by Sbrenthughes** [comment]
Can you paste a screenshot of your product dashboard?

**Comment by sandilyasd** [comment]
![image](https://user-images.githubusercontent.com/10300329/46477934-7780f100-c809-11e8-8fd7-c5bf677677dd.png)

**Comment by sandilyasd** [comment]
Yes, the build and Repo has the same repo configured. But the there might be multiple branches configured.

And also i have opened the Hygiea in the private window.

**Comment by Sbrenthughes** [comment]
I am wondering if Subversion even pushes the correct data required for that. I will see what I can find.

**Comment by vivekshm** [comment]
We started CI/CD only for Dev environment intially.

We have Hygieia 3.0.2 version, NPM version 6.4.1 and Node 10.11.1.

We have SVN for repo widget and Jenkins for build widget and both are configured in Team Dashboard using collectors and data is refelected in repo widget as soon as developer commits the code in SVN. But nothing is displayed in Commit stage and Build stage in product dashboard, however I can see data in Build stage.

Apart from that, I am using Hygieia publisher plug-in for deployment and changes in team dashboard reflected in deployment widget. The same SVN branch is configured in deployment job for build and deploy as in CI job in Jenkins.

We tried to look into pipeline table in mongodb, I can only see two stages Build and Dev (Deployment), seems commit stage is missing.

We also tried to clear the browser data with no success.

Can anyone please help me on this.
![dashboard](https://user-images.githubusercontent.com/57887662/69028447-034ae200-09f8-11ea-9803-e007d6ff1ef8.JPG)
